src/Native/System.Security.Cryptography.Native/pal_cms.h

```diff
@@ -61,3 +61,54 @@ extern "C" int CryptoNative_CmsGetRecipientStackFieldCount(CmsRecipientStack* re
 Shims the CMS_RecipientInfo_free method
 */
 extern "C" void CryptoNative_CmsRecipientDestroy(CMS_RecipientInfo* recipient);
+
+/*
+Initializes a CMS_ContentInfo of type EnvelopedData with the cipher given by the oid in algorithmOid
+and returns a pointer to it. 
+
+Status is set to 1 on success, 0 on unsupported algorithm, and -1 on invalid input.
+*/
+extern "C" CMS_ContentInfo* CryptoNative_CmsInitializeEnvelope(ASN1_OBJECT* algorithmOid, int32_t* status);
+
+/*
+Adds an originator certificate to the CMS_ContentInfo structure. If the CMS_ContentInfo structure
+is of type EnvelopedData it will be added to OriginatorInfo, and for SignedData it will be added to
+the certificates.
+
+Returns 1 on success, 0 on OpenSSL failure, and -1 on invalid input.
+*/
+extern "C" int CryptoNative_CmsAddCert(CMS_ContentInfo* cms, X509* cert);
```

Comment:
I'm still not seeing an implementation function that this maps to; and not seeing CmsAddOriginatorCert in the header.  Am I missing something?